### ISSUE TITLE
Perceiver IO Learning Rate

### ISSUE THREAD

**Issue by ShoufaChen** [deepmind/deepmind-research#278]
Hi,

Thanks for your great work.

I saw the learning rate here:

https://github.com/deepmind/deepmind-research/blob/769bfdbeafbcb472cb8e2c6cfa746b53ac82efc2/perceiver/train/experiment.py#L88

The paper said the 'a flat learning rate of 2e-3'.

I was also wondering which value is correct?

**Comment by ShoufaChen** [comment]
Thanks for pointing this out.
